models/action.go

```diff
@@ -706,5 +741,12 @@ func GetFeeds(opts GetFeedsOptions) ([]*Action, error) {
 	if opts.RequestedUser.IsOrganization() {
 		sess.In("repo_id", repoIDs)
 	}
+
+	if !opts.IncludeDeletedComments {
+		// commentID != 0 AND comment exist
+		sess.And("(comment_id ISNULL OR comment_id = 0 OR EXISTS(SELECT NULL FROM comment c WHERE comment_id = c.id))")
```

Comment:
Select is invalid as there is no such `ISNULL`. Also selects with `OR` are very hard to optimize for dbms so mostly they are slow on large tables and this one is such. I would better prefer if there would be added new `bool` field that action has been deleted that would be updated to true on deleting comment etc